css/banners.css

```diff
@@ -1,31 +1,13 @@
-@import url(http://db.onlinewebfonts.com/c/0dabf6820744e4ed41f7ce1dd8a9e502?family=SackersGothicW01-Light);
-@import url(http://db.onlinewebfonts.com/c/65bc711a745ac850f48c5e33d3e3e58c?family=DidotW01-Roman);
-@font-face {
-    font-family: "SackersGothicW01-Light";
-    src: url("http://db.onlinewebfonts.com/t/0dabf6820744e4ed41f7ce1dd8a9e502.eot");
-    src: url("http://db.onlinewebfonts.com/t/0dabf6820744e4ed41f7ce1dd8a9e502.eot?#iefix") format("embedded-opentype"),
-    url("http://db.onlinewebfonts.com/t/0dabf6820744e4ed41f7ce1dd8a9e502.woff") format("woff"),
-    url("http://db.onlinewebfonts.com/t/0dabf6820744e4ed41f7ce1dd8a9e502.ttf") format("truetype"),
-    url("http://db.onlinewebfonts.com/t/0dabf6820744e4ed41f7ce1dd8a9e502.svg#SackersGothicW01-Light") format("svg");
-}
-@font-face {font-family: "DidotW01-Roman";
-    src: url("http://db.onlinewebfonts.com/t/65bc711a745ac850f48c5e33d3e3e58c.eot");
-    src: url("http://db.onlinewebfonts.com/t/65bc711a745ac850f48c5e33d3e3e58c.eot?#iefix") format("embedded-opentype"),
-    url("http://db.onlinewebfonts.com/t/65bc711a745ac850f48c5e33d3e3e58c.woff") format("woff"),
-    url("http://db.onlinewebfonts.com/t/65bc711a745ac850f48c5e33d3e3e58c.ttf") format("truetype"),
-    url("http://db.onlinewebfonts.com/t/65bc711a745ac850f48c5e33d3e3e58c.svg#DidotW01-Roman") format("svg");
-}
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+@import url(https://fonts.googleapis.com/css?family=Libre+Baskerville:400,400italic);
```

Comment:
can you delete these two lines, `git pull --rebase upstream master` or it's equivalent, and verify that the fonts still work? `@import 'common'` should take care of this for you.